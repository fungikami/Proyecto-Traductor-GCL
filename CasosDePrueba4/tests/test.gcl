|[
    declare
        x : int;
        y : bool

    skip;
    |[
        declare 
            z : int
        skip
    ]|
]|