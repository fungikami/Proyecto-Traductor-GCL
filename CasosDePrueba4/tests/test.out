c_{34} (c_{34} (c_{39}(c_{24}(c_{20} (c_{40}))(c_{32} (c_{32} c_{36} c_{37}) c_{36}))) (c_{67})) (c_{39}(c_{24}(c_{20} (c_{40}))(c_{32} c_{37} c_{36})))